scripts/build_native.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cmake -B native/build/ -S native/ -GNinja &&
+ninja -C native/build/  &&
+./native/build/consteval
```

Comment:
Huh, didn't know you don't need `\` at the end of your line ends with `&&`.
Live and learn.